specification_documents/examples/intro_qc2.mzQC

```diff
@@ -11,7 +11,7 @@
           "inputFiles": [
             {
               "location": "file://tmp/QC2_18052020.mzML",
-              "name": "QC type 2 sample",
+              "name": "QC2_18052020",
```

Comment:
**question:** Can the raw file and the ID file both have the same `name` attribute?